app/controllers/artefacts_controller.rb

```diff
@@ -48,7 +48,6 @@ def new
   end
 
   def edit
```

Comment:
Yes, you can still edit it in this app. It just doesn't need a special ivar set (`@artefact` is set in a before action method).